src/openrct2-ui/windows/EditorBottomToolbar.cpp

```diff
@@ -182,27 +182,6 @@ class EditorBottomToolbarWindow final : public Window
         GfxInvalidateScreen();
     }
 
-    bool CheckObjectSelection() const
```

Comment:
I moved this to another header file so I could import it in the EditorObjectSelection file.